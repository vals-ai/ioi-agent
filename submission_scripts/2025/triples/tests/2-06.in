version https://git-lfs.github.com/spec/v1
oid sha256:0210d32101bc4475c8ceebab7d950b952d0c229ba7a1aca1aed6701bacb79451
size 433296
