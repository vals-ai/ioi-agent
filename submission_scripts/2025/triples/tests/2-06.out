version https://git-lfs.github.com/spec/v1
oid sha256:818353b793606084c6caa1dad9539521e54e38987e41ec2b7f680bce12d57cef
size 41
